app/web/src/components/Svg/Svg.tsx

```diff
@@ -331,21 +331,21 @@ const Svg = ({
       </svg>
     ),
     gear: (
-      <svg viewBox="0 0 19 18" fill="none" xmlns="http://www.w3.org/2000/svg">
+      <svg viewBox="0 0 33 33" fill="none" xmlns="http://www.w3.org/2000/svg">
         <path
           fillRule="evenodd"
           clipRule="evenodd"
           fill="currentColor"
-          d="M8.86193 0.104004H10.1383L10.964 2.58138L11.506 2.87613L13.2571 1.19177L14.3466 1.85664L13.761 4.40148L14.174 4.69029L16.7384 4.1958L17.3641 5.30824L15.4304 7.10054L15.5487 7.49835L18.0261 8.32463V9.601L15.5487 10.4267L15.4304 10.836L17.3641 12.617L16.7384 13.7294L14.174 13.2349L13.761 13.5237L14.3466 16.0686L13.2571 16.7335L11.506 15.1591L10.964 15.4188L10.1383 17.8962H8.86193L8.03564 15.4187L7.51868 15.1591L5.74267 16.7335L4.65315 16.0686L5.23879 13.5237L4.82575 13.2349L2.2614 13.7294L1.63562 12.617L3.57334 10.836L3.45101 10.4267L0.973633 9.601V8.32463L3.45107 7.49835L3.57334 7.10054L1.63562 5.30824L2.2614 4.1958L4.82575 4.69029L5.23879 4.40148L4.65315 1.85664L5.74267 1.19177L7.51868 2.87613L8.03564 2.58144L8.86193 0.104004ZM9.49979 11.7199C11.0019 11.7199 12.2195 10.5022 12.2195 9.00009C12.2195 7.49801 11.0019 6.28033 9.49979 6.28033C7.99771 6.28033 6.78003 7.49801 6.78003 9.00009C6.78003 10.5022 7.99771 11.7199 9.49979 11.7199Z"
-        />
+          d="M18.5596 32.6494L14.4414 32.6493L13.3126 27.3566L11.1836 26.4752L6.6429 29.4195L3.73084 26.5076L6.75991 22.0513L5.79279 19.838L0.500103 18.7091L0.5 14.591L5.79292 13.5818L6.75988 11.2476L3.73082 6.79131L6.64287 3.8794L11.1836 6.82369L13.3115 5.9422L14.4403 0.649517L18.5585 0.649414L19.5677 5.94233L21.9012 6.90846L26.3575 3.87939L29.2694 6.79145L26.3251 11.3321L27.2071 13.5807L32.5 14.5899L32.4999 18.708L27.2072 19.8369L26.3251 21.9667L29.2694 26.5074L26.3575 29.4195L21.9012 26.3904L19.5688 27.3565L18.5596 32.6494ZM16.5001 22.2987C13.3792 22.2987 10.8492 19.7687 10.8492 16.6478C10.8492 13.5269 13.3792 10.997 16.5001 10.997C19.621 10.997 22.151 13.5269 22.151 16.6478C22.151 19.7687 19.621 22.2987 16.5001 22.2987Z"/>
       </svg>
     ),
     'big-gear': (
```

Comment:
If the `IdeSidebar` is the only place that `big-gear` is used we can probably do away with it; they're the same SVG now.

Reply:
Yup 👍 , I added it for the side bar so shouldn't be used anywhere else.